### DISCUSSION TITLE
Update to theme dark mode appearance

### DISCUSSION THREAD

**Discussion by bnhassin** [community/community#22162] [Accessibility]
<p><strong>Feedback Theme Beta with Screen</strong></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/d/ed8b22388c9d726c2a7f88f3cba949b65a376b61.png" data-download-href="/uploads/short-url/xTpfGC1a3guPOtAZCDXrkS8ExZn.png?dl=1" title="Screenshot_20201213-211310" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1473765/181069283-fb5c565f-f7ca-47da-8cdc-4d6b93afc06b.png" alt="Screenshot_20201213-211310" data-base62-sha1="xTpfGC1a3guPOtAZCDXrkS8ExZn" width="250" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed8b22388c9d726c2a7f88f3cba949b65a376b61_2_250x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed8b22388c9d726c2a7f88f3cba949b65a376b61_2_375x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/d/ed8b22388c9d726c2a7f88f3cba949b65a376b61.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed8b22388c9d726c2a7f88f3cba949b65a376b61_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_20201213-211310</span><span class="informations">480×960 51.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<aside class="quote quote-modified" data-post="1" data-topic="149629">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/update-to-theme-dark-mode-appearance/149629">Update to theme dark mode appearance</a> <a class="badge-wrapper  bullet" href="/c/github-help/dark-mode/65"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #0088CC;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Dark mode is a new color theme joining the classic GitHub light mode developers have been using for years. We’re excited that dark mode has (finally) arrived in a public beta.">Dark Mode</span></a>
  </div>
  <blockquote>
    Feedback Theme Beta with Screen 
[Screenshot_20201213-211310] 
<a href="https://github.community/t/update-to-theme-dark-mode-appearance/149629" class="onebox" target="_blank" rel="noopener">https://github.community/t/update-to-theme-dark-mode-appearance/149629</a>
  </blockquote>
</aside>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/bnhassin">@bnhassin</a> - thanks for reaching out! Can you help me understand the feedback you are sharing?</p>

**Comment by bnhassin** [comment]
<p>Hi <a class="mention" href="/gaknoll">@gaknoll</a></p>
<p>Setting my GitHub profile appearance to dark mode theme</p>

**Comment by bnhassin** [comment, marked as answer]
<p>I do for enabled theme settings profile from the direct <a href="https://github.com/settings/appearance" rel="noopener nofollow ugc">link</a><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/6/8616211effbd5aa75f340d320c7b57bddad561e8.png" data-download-href="/uploads/short-url/j8bomKQSK98cz6vszSBCDCMfCAg.png?dl=1" title="Screenshot_20201222-221043" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1473765/181069287-9c19ab06-83e9-46d4-93ef-74f71ffb46ae.png" alt="Screenshot_20201222-221043" data-base62-sha1="j8bomKQSK98cz6vszSBCDCMfCAg" width="250" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8616211effbd5aa75f340d320c7b57bddad561e8_2_250x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8616211effbd5aa75f340d320c7b57bddad561e8_2_375x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/6/8616211effbd5aa75f340d320c7b57bddad561e8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8616211effbd5aa75f340d320c7b57bddad561e8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20201222-221043</span><span class="informations">480×960 75.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by cehfisher** [comment]
Hi @bnhassin - this issue seems to have been resolved. Could you please confirm and close this issue? Thanks! (cc @queenofcorgis)

**Comment by bnhassin** [comment]
**[#22162](https://github.com/community/community/discussions/22162)**
